packages/ui/src/kit/VDateInput/VDateInput.vue

```diff
@@ -0,0 +1,228 @@
+<template>
+  <div ref="root" :class="vDateInputCssClass">
+    <div class="v-date-input-field">
+      <input-atom
+        v-bind="$attrs"
+        :value="formattedValue"
+        @focus="onFocusInput"
+        @blur="onBlurInput"
+      />
+      <div class="v-date-input-field-icon">
+        <v-svg-icon name="calendar" />
+      </div>
+    </div>
+    <div v-if="isCalendarVisible" class="v-date-input-calendar">
```

Comment:
layout have wrong struct again ( doh...

Reply:
?

Reply:
`class` and `v-if` must be in the next line, code style for attributes in tag fault.

Reply:
Ah, it is questions to our tooling 🙃